### PR TITLE
Add demo for GoJS.

### PR DESCRIPTION
**Pull request by mn98**

### CODE REVIEW

**Comment by oyvindberg** [comment]
Can you change base to point to `master`, so we use this PR for the whole demo?

**Comment by mn98** [comment]
> Can you change base to point to `master`, so we use this PR for the whole demo?

Done!

File: gojs/src/main/js/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>GoJS Demo</title>
+    <style>
```
Comment (mn98) [inline gojs/src/main/js/index.html:6]: I think it's a little unfortunate that this style is specified in html. How would we move this into the slinky component?

Comment (oyvindberg) [reply gojs/src/main/js/index.html:6]: Well, the interface of the `ReactDiagram` component requires a global css class name, so to follow the spirit of that you can use any way of loading css globally, which is a broad topic.

That said, maybe you can do something like this. 

```scala
import typings.react.components.div
import typings.react.mod.CSSProperties

div.style(CSSProperties().setWidth("400px").setHeight("400px"))(
    ReactDiagram(
      divClassName = "non-existing",
      initDiagram = diagram,
      nodeDataArray = nodes,
      skipsDiagramUpdate = false
    )
      .linkDataArray(links)
      .onModelChange(_ => ())
)
```

And then see if the styling you set on the wrapper div is inherited by whatever the diagram draws. You can use some slinky syntax for this, but I only remember how to use this and cant check right now

Comment (mn98) [reply gojs/src/main/js/index.html:6]: Sadly, it doesn't inherit the style from the parent div. We just get an empty 400x400 div containing another div without the appropriate styling which contains the canvas.

Comment (oyvindberg) [reply gojs/src/main/js/index.html:6]: Aha, too bad. In that case let's leave the demo as is, this is a quite common problem in the javascript world that you need to load some global css things before things work.

File: build.sbt [hunk 2]
```diff
@@ -371,6 +371,19 @@ lazy val `react-native` = project
     }
   )
 
+lazy val gojs = project
+  .enablePlugins(ScalablyTypedConverterPlugin)
+  .configure(baseSettings, browserProject, reactNpmDeps, bundlerSettings)
+  .settings(
+    useYarn := true,
+    webpackDevServerPort := 8010,
```
Comment (oyvindberg) [inline build.sbt]: can you pick the next available number? from the readme it looked like it already went to 8018

Comment (mn98) [reply build.sbt]: Done, I've set it to 8019.

**Comment by oyvindberg** [comment]
Oh, and as a last thing - could you run gojs/dist in sbt and check in the files please? I'll merge right after :)

**Comment by mn98** [comment]
> Oh, and as a last thing - could you run gojs/dist in sbt and check in the files please? I'll merge right after :)

Done :-)

**Comment by oyvindberg** [comment]
Thanks! :)

**Pull request merged**